### PR TITLE
feat: Text_Utilities and lists

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Added TypoGraphy and Lists 

Typography: Headings , Normal Text and Grey Text
Lists: Basic Lists and  Stacked(Notification) List

### CODE REVIEW

**Review by biswanath-tewari — APPROVED** [review]
looks all good to me.

**Review by ozairkhan2810 — APPROVED** [review]
looks good, can add center text else merge

**Comment by munjalSaksham71** [comment]
> looks good, can add center text else merge

Okay cool, I do it now,

File: utils/lists.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.list {
+    display: flex;
+    flex-direction: column;
+}
+
+.basic-list {
+    border: 1px solid black;
+    padding: 16px;
```
Comment (paarthbajaj) [inline utils/lists.css]: Go constant with the units please and prefer rem.

File: utils/lists.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.basic-list:hover {
+    cursor: pointer;
+    background-color: var(--primary);
+    color: #ffffff;
```
Comment (paarthbajaj) [inline utils/lists.css]: It is good that you have used CSS variable for color in line 13, it will be good if you do it same across all the CSS files.

**Review by paarthbajaj — APPROVED** [review]
Please see my comments then you can merge.

**Pull request merged**